uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
+  </header>
+  <div class="table-responsive">
+    <table class="table">
+      <tr>
```

Comment:
The column header should be part of `thead, th` tags. Refer https://github.com/civicrm/civihr/blob/016ea3a20c792d2a1759e79429a49cd4cffe79a5/uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-work-patterns.html#L9-L9

If you do that, we wont need the `<strong>` anymore.